README.md

```diff
@@ -86,6 +86,15 @@ $invoice->issue();
 $invoice->notifyBy('email');
 $invoice->cancel();
 $invoice->delete();
+
+// Virtual Accounts
+$virtualAccount  = $api->virtualAccount->create(array('receiver_types' => array('bank_account'), 'description' => 'First Virtual Account', 'notes' => array('receiver_key' => 'receiver_value')));
+$virtualAccount  = $api->virtualAccount->fetch('va_4xbQrmEoA5WJ0G');
+$virtualAccounts = $api->virtualAccount->all();
+$virtualAccount  = $api->virtualAccount->close('va_4xbQrmEoA5WJ0G');
```

Comment:
@harman28 should this not have been:
```
$api->virtualAccount->fetch('va_4xbQrmEoA5WJ0G')->close();
```

Going by other functions - capture, reverse, cancel.

Reply:
Yes, this makes sense.

Reply:
Updated